integration

```diff
@@ -198,6 +198,9 @@
   "custom-components/sensor.nintendo_wishlist",
   "custom-components/sensor.owlintuition",
   "custom-components/sensor.personalcapital",
+  "custom-components/sensor.plex_recently_added",
+  "custom-components/sensor.radarr_upcoming_media",
+  "custom-components/sensor.sonarr_upcoming_media",
```

Comment:
Changes more than one thing.

Reply:
My appolgies.  So, I should create a single PR for each item?  I will close this PR and do it correctly